### DISCUSSION TITLE
No response after scanning

### DISCUSSION THREAD

**Discussion by LMMinThere** [WalletConnect/walletconnect-monorepo#2124] [Developer Support] [answered]
Hi， I'm building a website and want it to support walletconnect login. I refer to the official documentation of V2 to write the code, and I can already get the QR code, including topic and uri. Then my work got stuck here, and there was no way to respond after scanning the code. Our project has not been approved by walletconnect yet, I don't know if this is the reason for the error, hope to get an answer

**Comment by amcs99** [comment]
Same for me. Have you got the solution?

**Comment by xzilja** [comment]
Are the wallet you are scanning with supporting v2 protocol?

**Reply by formatCvt** [reply]
Same issue here.

> Are the wallet you are scanning with supporting v2 protocol?

i think so, I'm trying with metamask for Android version 6.2.0

**Comment by queenandme28** [comment]
Same issue here. Metamask for Android. No response after scanning qr code.

**Comment by xzilja** [comment, marked as answer]
MetaMask does not support v2 yet, work is in progress

**Comment by amcs99** [comment]
For me, it's caused by I config Wallet Connect  `version 2`
`const wagmiClient = createClient({
    autoConnect: false,
    connectors: w3mConnectors({ chains, version: 2, projectId }),
    provider,
});`
But MetaMask wallet hasn't support Wallet Connect version 2 yet(see the list wallet that supported Wallet Connect version 2 [here](https://explorer.walletconnect.com/?type=wallet&version=2) ). I just changed version 2 to 1 and it worked properly
`const wagmiClient = createClient({
    autoConnect: false,
    connectors: w3mConnectors({ chains, version: 1, projectId }),
    provider,
});`

Also, hope that MetaMask will support WC version 2 before this summer 😆
